File: Front-End/ecommercePFE/src/app/model/UserModel.ts
export class UserModel{
    private email: string;
    private mdp: string;
    private nom:string;
    private prenom:string;
    private confirmMdp:string;
    private role:string;
    private region:string;
    private numTel:string;
    private codePostal:string;
    private adresse:string;

    constructor(email:string, mdp:string, nom:string, prenom:string,confirmMdp:string,role:string){
        this.email=email;
        this.mdp=mdp;
        this.nom=nom;
        this.prenom=prenom;
        this.confirmMdp=confirmMdp;
        this.role=role;
    }

    setEmail(email:string):void{
        this.email=email;
    }

    getEmail(): string{
        return this.email;
    }

    getMdp(): string{
        return this.mdp;
    }
    getPrenom(): string{
        return this.prenom;
    }
    getNom(): string{
        return this.nom;
    }

    getConfirmMdp(){
        return this.confirmMdp;
    }

    getCodePostal(){
        return this.role;
    }
    getNumTel(){
        return this.role;
    }
    getAdresse(){
        return this.role;
    }
    getRegion(){
        return this.role;
    }
    getRole(){
        return this.role;
    }

    




}

File: Back-End/Tunishop/src/main/java/com/example/Tunishop/Model/UserModel.java
package com.example.Tunishop.Model;



import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserModel {
	
	private String email;
	private String nom;
	private String prenom;
	private String mdp;
	private String confirmMdp;
	private String role;
	private String region;
	private String codePostal;
	private String adresse;
	private String numTel;
	
	//GETTERS && SETTERS
	
	
	
	public String getRegion() {
		return region;
	}
	public void setRegion(String region) {
		this.region = region;
	}
	
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getNom() {
		return nom;
	}
	public void setNom(String nom) {
		this.nom = nom;
	}
	public String getPrenom() {
		return prenom;
	}
	public void setPrenom(String prenom) {
		this.prenom = prenom;
	}
	public String getMdp() {
		return mdp;
	}
	public void setMdp(String mdp) {
		this.mdp = mdp;
	}
	public String getConfirmMdp() {
		return confirmMdp;
	}
	public void setConfirmMdp(String confirmMdp) {
		this.confirmMdp = confirmMdp;
	}
	public String getRole() {
		return role;
	}
	public void setRole(String role) {
		this.role = role;
	}
	public String getCodePostal() {
		return codePostal;
	}
	public void setCodePostal(String codePostal) {
		this.codePostal = codePostal;
	}
	public String getAdresse() {
		return adresse;
	}
	public void setAdresse(String adresse) {
		this.adresse = adresse;
	}
	public String getNumTel() {
		return numTel;
	}
	public void setNumTel(String numTel) {
		this.numTel = numTel;
	}
	
	
	
	
}
